### ISSUE TITLE
No push notification

### ISSUE THREAD

**Issue by Tealk** [nucleus-ffm/foss_warn#73]
Hello,

I guess since some updates of the app I don't get any notification anymore.
Pixel 6
CalyxOS 4.3.2
Push provider ntfy

I noticed because today should be warning day, the app shows the warning but also again the message "No Internet".

![Screenshot_20221208-111629.png](https://user-images.githubusercontent.com/12276250/206421109-27a07bc3-ef0a-4029-a569-23c92945bee0.png)

**Comment by realpixelcode** [comment]
Yeah, I haven't received any notifications from FOSS Warn today either. I guess the government's MoWaS (Modular Warning System) was overloaded just like two years ago on the last Alert Day. However, that doesn't explain why FOSS Warn didn't issue ANY notification at all, since the official NINA warning app did, in fact, do so (just with a short delay).

**Comment by Tealk** [comment]
The "No Internet" is also gone again since the second warning message

**Comment by nucleus-ffm** [comment]
In the time between 11 and 11:10 I also noticed that the server is not responding. Of course, the message "no internet" is only related to whether the server is available or not. 
Last time someone was monitoring the servers of NINA during the warning day, but unfortunately this time I didn't find anything. It would be interesting to know in what time exactly the server was unreachable. If someone has an idea who did that last time or if someone did that this time as well, I would be happy to hear about it.

**Comment by realpixelcode** [comment]
Perhaps change the error message to something like “Warning server unreachable”? And possibly also ping other domains as well to learn whether it's the device that is offline or the warning server?
